lib/enum/basic_enum.rb

```diff
@@ -22,48 +22,58 @@ def number_of_elements_in_collection
   # This one also uses 'count'
   # Try it yourself!
   def number_of_floats_or_fixnums
+    collection.count{|x| (x.is_a? Fixnum) || (x.is_a? Float)}
```

Comment:
Make sure your local variables have intuitive names. In this case, that might be `|element|` or `|item|`.